apps/busybox/deployers/busybox_statefulset.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    lkey: lvalue
+  name: busybox
+spec:
+  clusterIP: None
+  selector:
+    lkey: lvalue
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: busybox
+  labels:
+    lkey: lvalue
+spec:
+  serviceName: busybox
+  rkey: rvalue
+  selector:
+    matchLabels:
+      lkey: lvalue
+      openebs.io/replica-anti-affinity: lvalue
```

Comment:
@nsathyaseelan , Can we have a comment here, saying that this label needs to be added to application in order to distribute replicas.